templates/allauth/account/email/email_confirmation_message.txt

```diff
@@ -0,0 +1,9 @@
+{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}Hello from {{ site_name }}!
```

Comment:
Could you maybe split this into multiple lines?

Reply:
These are the default allauth templates that I mentioned the other day. They come in like that, but yep, I will sort it into multi-line.